@leoyli getting some linting errors:

```
/tmp/storybook/addons/knobs/src/components/types/Object.js
  12:7  error  Unused state field: 'values'  react/no-unused-state
  14:7  error  Unused state field: 'state'   react/no-unused-state
```

--- a/addons/knobs/src/components/types/Object.js
+++ b/addons/knobs/src/components/types/Object.js
@@ -5,15 +5,11 @@ import { polyfill } from 'react-lifecycles-compat';
 import { Form } from '@storybook/components';
 
 class ObjectType extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      value: null,
-      failed: false,
-      json: props.knob.value,
-    };
-  }
+  state = {
+    value: {},
+    failed: false,
+    json: '',
+  };
 
   static getDerivedStateFromProps(props, state) {
     if (!deepEqual(props.knob.value, state.json)) {

--- a/addons/knobs/src/components/types/Object.js
+++ b/addons/knobs/src/components/types/Object.js
@@ -9,9 +9,9 @@ class ObjectType extends Component {
     super(props);
 
     this.state = {
-      values: null,
+      value: null,
       failed: false,
-      state: '',
+      json: props.knob.value,
     };
   }
 

